src/Umbraco.Web/Trees/ApplicationTreeController.cs

```diff
@@ -71,9 +86,64 @@ public async Task<SectionRootNode> GetApplicationTrees(string application, strin
                 }
             }
 
-            var multiTree = SectionRootNode.CreateMultiTreeSectionRoot(rootId, collection);
-            multiTree.Name = Services.TextService.Localize("sections/"+ application);
-            return multiTree;
+            //Don't apply fancy grouping logic futher down, if we are not 'settings' section
+            if(application != Constants.Applications.Settings)
```

Comment:
Do we need this check? Shouldn't we just check if there are no groupings for the current section for any trees and deal with it accordingly? That way in the future groups will 'just work' for any tree that has groupings appplied